src/components/UserInfo.js

```diff
@@ -1,18 +1,23 @@
 export default class UserInfo {
-    constructor(nameSelector, jobSelector) {
+    constructor(nameSelector, jobSelector, userAvatar) {
         this._nameElement = document.querySelector(nameSelector);
         this._jobElement = document.querySelector(jobSelector);
+        this._avatar = document.querySelector(userAvatar);
       }
-    
+
       getUserInfo() {
         return {
           name: this._nameElement.textContent,
           description: this._jobElement.textContent,
         };
       }
-    
+
       setUserInfo(userData) {
         this._nameElement.textContent = userData.name;
-        this._jobElement.textContent = userData.description;
+        this._jobElement.textContent = userData.about;
+      }
+
+      setAvatar({ avatar }) {
```

Comment:
It's great to have a setter for the avatar as it's controlled by a different form.